bundles/net.tourbook.chart/src/net/tourbook/chart/ChartComponentGraph.java

```diff
@@ -6364,13 +6310,10 @@ private ChartYSlider isYSliderHit(final int graphX, final int devY) {
          final boolean canDoHitChecking = !_canChartBeOverlapped
                || (_canChartBeOverlapped && (isStackedChart || isLastOverlappedGraph));
 
-         if (canDoHitChecking) {
-
-            if (ySlider.getHitRectangle().contains(graphX, devY)) {
+         if (canDoHitChecking && ySlider.getHitRectangle().contains(graphX, devY)) {
 
-               _hitYSlider = ySlider;
-               return ySlider;
-            }
```

Comment:
Sometimes this complex code in the tour chart is not "optimized" to do better debugging, e.g. you cannot set in the optimized code a breakpoint to the old line 6369, this mostly effects also ALL THE OTHER OPTIMIZATIONS

Reply:
Reverted